web/src/OverviewResourceSidebar.stories.tsx

```diff
@@ -71,6 +71,11 @@ export const ResourcesWithLabels = () => {
       labels["test"] = "test"
     }
 
+    if (i === 3) {
```

Comment:
I don't know if this is a new issue in this PR, and not really a blocker, but if it's cheap to fix: this surfaces that the status icon for this group is a bit unaligned with the status icons for the other groups:
![image](https://user-images.githubusercontent.com/7453991/125839262-33064cbb-0a49-4432-add8-28abaf5daa41.png)